redaxo/src/core/lib/login/backend_password_policy.php

```diff
@@ -42,25 +42,33 @@ class rex_backend_password_policy extends rex_password_policy
      */
     public function __construct()
     {
+        /**
+         * @var array<string, array<string, scalar>|string|int> $options
+         */
```

Comment:
```suggestion
        /** @var array<string, array<string, scalar>|string|int> $options */
```

Bei Variablen nutzen wir eigentlich immer die einzeilige Variante.